cli/version.rs

```diff
@@ -1,6 +1,13 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+use serde_json;
 pub const DENO: &str = env!("CARGO_PKG_VERSION");
 
 pub fn v8() -> &'static str {
   deno::v8_version()
 }
+
+pub fn typescript() -> String {
+  let data = include_str!("../node_modules/typescript/package.json");
```

Comment:
The only thing that bothers me about this is that we're unnecessarily embedding the package.json in our binary and decoding it at runtime. Can you add a TODO here:
```
// TODO By using include_str! we are including the package.json into the deno binary
// using serde to decode it at runtime. This is suboptimal in space and time. We need
// extract the TypeScript version at compile time instead. This will be easier after #2608.
```

Reply:
Added the comments.

BTW do you think extracting package.json in BUILD.gn is a bad idea? (Then pass the version number as an env var like `CARGO_PKG_VERSION`)

Reply:
yes and no. 

It’s a good idea to do this at compile time, but we’re moving away from GN builds. Soon we’ll remove build.py and define most of the stuff that was previously in BUILD.gn in a build.rs file.  So I think once that’s done we can basically move your json parsing function into cli/build.rs